third_party/blink/renderer/brave_font_whitelist.cc

```diff
@@ -9,13 +9,22 @@
 #include <vector>
 
 #include "base/containers/fixed_flat_set.h"
+#include "base/linux_util.h"
 
 namespace brave {
 
 namespace {
 
 bool g_simulate_empty_font_whitelist_for_testing = false;
 
+#if BUILDFLAG(IS_LINUX)
+const auto g_distro = base::GetLinuxDistro();
```

Comment:
this is not great. You don't need this global variable at the very start of the process. Consider a more startup-friendly approach:
```
const std::string& GetLinuxDistro() {
  static const base::NoDestructor<std::string> linux_distro(base::GetLinuxDistro());
  return *linux_distro;
}
```

Reply:
For some reason when I don't call this function early in the process, it always returns "Unknown" for the Linux distro. Whereas if I call it at process start, then I get the correct string "Ubuntu 22.04.3 LTS". So far I'm not sure why this happens, but I'm wondering if you think an early call could be acceptable under the circumstances.

Reply:
Addressed in https://github.com/brave/brave-core/pull/20930#discussion_r1395218706